bindings/python/docs/source/supported_types.rst

```diff
@@ -27,6 +27,8 @@ Support for additional types will be added in subsequent releases.
      - :class:`~py.int`, :class:`bson.int64.Int64`, an instance of :meth:`pyarrow.int64`
    * - UTC datetime
      - an instance of :class:`~pyarrow.timestamp` with ``ms`` resolution, :class:`py.datetime.datetime`
+   * - Boolean
+     - an instance of :class:`~pyarrow.bool_`, :class:`~py.bool`
```

Comment:
I've been putting these in the same order as their [type number](https://docs.mongodb.com/manual/reference/bson-types/)

Reply:
Ah ok, makes sense. Done.